src/libraries/Common/src/System/Net/Http/Http3/QPack/QPackDecoder.cs

```diff
@@ -222,7 +223,7 @@ private void OnByte(byte b, IHttpHeadersHandler handler)
                 case State.CompressedHeaders:
                     switch (BitOperations.LeadingZeroCount(b))
                     {
-                        case 0: // Indexed Header Field
+                        case 24: // Indexed Header Field
```

Comment:
@scalablecory I think both of us fixed this in our PRs. Taking either fix is okay I think.

Reply:
I changed it to your fix.